test/spec/GeoExt/data/store/WfsFeatures.test.js

```diff
@@ -0,0 +1,166 @@
+Ext.Loader.syncRequire(['GeoExt.data.store.WfsFeatures']);
+
+describe('GeoExt.data.store.WfsFeatures', function() {
+
+    describe('basics', function() {
+        it('is defined', function() {
+            expect(GeoExt.data.store.WfsFeatures).not.to.be(undefined);
+        });
+    });
+
+    describe('constructor (no arguments)', function() {
+
+        it('raises an error', function() {
+            var errRaised = false;
+            try {
+                Ext.create('GeoExt.data.store.WfsFeatures');
+            } catch (e) {
+                errRaised = true;
+            }
+
+            expect(errRaised).to.be(true);
+        });
+    });
+
+    describe('configs and properties', function() {
+        var store;
+        beforeEach(function() {
+            store = Ext.create('GeoExt.data.store.WfsFeatures', {
+                url: 'foo'
+            });
+        });
+        afterEach(function() {
+            store.destroy();
+        });
+
+        it('are correctly defined (with defaults)', function() {
+            expect(store.service).to.be('WFS');
+            expect(store.version).to.be('2.0.0');
+            expect(store.request).to.be('GetFeature');
+            expect(store.typeName).to.be(null);
+            expect(store.outputFormat).to.be('application/json');
+            expect(store.sortBy).to.be(null);
+            expect(store.startIndex).to.be(0);
+            expect(store.count).to.be(null);
+            expect(store.startIndexOffset).to.be(0);
+        });
+    });
+
+    describe('loading without paging', function() {
+        var dataPath = (typeof __karma__ === 'undefined' ? '' : 'base/test/');
```

Comment:
Under which circumstances is `__karma__` undefined? I never thought this can happen :man_shrugging: